awx/api/views.py

```diff
@@ -1299,8 +1299,11 @@ def post(self, request, *args, **kwargs):
             if not project_update:
                 return Response({}, status=status.HTTP_400_BAD_REQUEST)
             else:
+                data = OrderedDict()
+                data.update(ProjectUpdateSerializer(project_update, context=self.get_serializer_context()).to_representation(project_update))
+                data['project_update'] = project_update.id
                 headers = {'Location': project_update.get_absolute_url(request=request)}
-                return Response({'project_update': project_update.id},
+                return Response(data,
```

Comment:
looks great, nice that you already had the data on-hand in the `project_update` object. The context allows the full serializer to be rendered.